Kafka_Consumer.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+
+import time
+import datetime
+import calendar
+import getopt
+import sys
+import os
+import yaml
+import socket
+import kafka
+import traceback
+
+from api import *
+
+def processMessage(msg):
+    """ Process the message
+
+    :param msg:     Message consumed
+
+    :return:
+    """
+
+    m = Message(msg.value) # Gets body of kafka message.
+    t = msg.topic # Gets topic of kafka message.
+
+    if t == "openbmp.parsed.router":
+        router = Router(m.getVersion(), m.getContent())
+        print t + '\n'
+        print router.toJsonPretty()
+
+    elif t == "openbmp.parsed.peer":
+        peer = Peer(m.getContent())
```

Comment:
let's have all the extend classes require "version" like router does.  This way we start it off requiring that. Alternatively, the first arg can be the message class object instead, which contains the version and content.